src/common/components/organisms/EditorPanel.tsx

```diff
@@ -1,33 +1,86 @@
-import React, { useCallback } from "react";
-import { RiPencilFill } from "react-icons/ri";
+import React from "react";
 import { ThemeSettings } from "@/common/lib/theme";
-import ThemeEditorToolbar from "@/common/lib/theme/ThemeEditorToolbar";
+import ThemeSettingsEditor from "@/common/lib/theme/ThemeSettingsEditor";
 import DEFAULT_THEME from "@/common/lib/theme/defaultTheme";
-import { mergeClasses } from "@/common/lib/utils/mergeClasses";
+import FidgetTray from "./FidgetTray";
 
 export interface EditorPanelProps {
-  editMode: boolean;
   setEditMode: (editMode: boolean) => void;
   theme?: ThemeSettings;
   saveTheme: (newTheme: ThemeSettings) => void;
-  disabled?: boolean;
+  unselect: () => void;
+  selectedFidgetID: string | null;
+  currentFidgetSettings: React.JSX.Element;
 }
 
 export const EditorPanel: React.FC<EditorPanelProps> = ({
-  editMode,
   setEditMode,
   theme = DEFAULT_THEME,
   saveTheme,
+  unselect,
+  selectedFidgetID,
+  currentFidgetSettings,
 }) => {
-  // const toggleEditMode = useCallback(() => {
-  //   !disabled && setEditMode(!editMode);
-  // }, [editMode, setEditMode, disabled]);
   return (
-    <ThemeEditorToolbar
-      theme={theme}
-      saveTheme={saveTheme}
-      setEditMode={setEditMode}
-    />
+    <div className="flex w-full">
+      <aside
+        id="editor-panel"
+        className="flex-col h-full z-8 w-7/12 transition-transform -translate-x-full sm:translate-x-0"
+        aria-label="Editor"
+      >
+        <div className="w-full h-full">
+          <div className="h-full px-4 py-4 overflow-y-auto border border-blue-100 rounded-xl relative bg-card">
+            <div className="flex-col">
+              {selectedFidgetID ? (
+                <>
+                  <div className="flex">
+                    <button onClick={unselect} className="my-auto">
+                      <BackArrowIcon />
+                    </button>
+                    <h1 className="capitalize text-lg pl-4">
+                      Edit {selectedFidgetID}
```

Comment:
This is fine for now, but Fidget ID will not be a nice human readable thing down the line

Reply:
Yeah we do need to replace this with a 'Fidget Name' field.